libraries/botbuilder-dialogs/package.json

```diff
@@ -50,7 +50,7 @@
     "build:rollup": "yarn clean && yarn build && api-extractor run --verbose --local",
     "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
     "depcheck": "depcheck --config ../../.depcheckrc",
-    "lint": "eslint .",
+    "lint": "eslint . --config ../../eslint.config.cjs",
```

Comment:
Remove the eslint-plugin-only-warn package. Also, if we keep the config file, the script command shouldn't change.

Reply:
fixed